test/e2e/tests/input-hidden.spec.js

```diff
@@ -11,8 +11,8 @@ describe('hidden thingy', function() {
 
     loadFixture('sample');
     browser.driver.executeScript('history.back()');
-    var expectedValue = browser.params.browser === 'safari' ? '{{ 7 * 6 }}' : '';
-    expect(element(by.css('input')).getAttribute('value')).toEqual(expectedValue);
+    //var expectedValue = browser.params.browser === 'safari' ? '{{ 7 * 6 }}' : '';
+    //expect(element(by.css('input')).getAttribute('value')).toEqual(expectedValue);
```

Comment:
we need to discuss this, this looks like we are skipping test that should be passing